src/Utils/SeoMetadataResolver.php

```diff
@@ -42,11 +42,25 @@ public function __construct(Title $title, Excerpt $excerpt, array $metadataConfi
      *
      * @param object|null $entity
      *
+     * @param bool $includePrefix
+     * @param bool $includeSuffix
      * @return string
      */
-    public function metaTitle(?object $entity): string
+    public function metaTitle(?object $entity, bool $includePrefix, bool $includeSuffix): string
     {
-        return $this->metadataConfig['title_prefix'].$this->meta($entity, 'title').$this->metadataConfig['title_suffix'];
+        if ($includePrefix && $includeSuffix) {
```

Comment:
You could simplify this bit by putting `$this->meta($entity, 'title')` in a variable, concatenating it as you go and eventually return the var as the last instruction.